ci-operator/config/openshift/aws-ebs-csi-driver/openshift-aws-ebs-csi-driver-master.yaml

```diff
@@ -23,7 +23,24 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14
   to: aws-ebs-csi-driver
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+    WORKDIR /go/src/github.com/openshift/build-machinery-go
+    RUN git clone -b add-commitchecker https://github.com/jsafrane/build-machinery-go.git .
+    RUN make -C commitchecker
+
+    FROM registry.ci.openshift.org/ocp/4.14:base
+    COPY --from=builder /go/src/github.com/openshift/build-machinery-go/commitchecker/commitchecker /usr/bin/
+    ENTRYPOINT ["/usr/bin/commitchecker"]
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.20-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14
+  to: commitchecker
 promotion:
+  excluded_images:
+  - commitchecker
```

Comment:
This will move to build-machinery-go CI job once https://github.com/openshift/build-machinery-go/pull/77 is merged.